con-base/src/main/java/ba/codecentric/base/domain/Hotel.java

```diff
@@ -163,11 +179,32 @@ public void setCountry(Country country) {
         this.country = country;
     }
 
-    @Override
-    public String toString() {
-        return "Id: " + this.getId() + ", Name: " + this.getName() + " , Rating: " + this.getRating() + " ,Address: " + this.getAddress() +
-                " ,Zip: " + this.getZip() + " ,City: " + this.getCity() +
-                " ,Phone: " + this.getPhone() + " ,Fax: " + this.getFax() + " ,Email: " + this.getEmail() + " ,Website: " + this.getWebsite() +
-                " ,Description: " + this.getDescription();
+    public Time getCheckIn() {
+        return checkIn;
+    }
+
+    public void setCheckIn(String checkIn) {
+        if(parseTime(checkIn)){
+            this.checkIn = Time.valueOf(checkIn + EXTEND_SECONDS);
+        } else  {
+            this.checkIn = null;
+        }
+    }
+
+    public Time getCheckOut() {
+        return checkOut;
+    }
+
+    public void setCheckOut(String checkOut) {
```

Comment:
This method has pretty much as same implementation as `setCheckIn`. Please refactor it and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)